### ISSUE TITLE
Does it support dynamic batch inference?

### ISSUE THREAD

**Issue by wangjingg** [maggiez0138/Swin-Transformer-TensorRT#2]
When I try onnx->tensorrt and use dynamic batch, the tensorrt output is all [0,0,0,0,.....]

**Comment by maggiez0138** [comment]
> When I try onnx->tensorrt and use dynamic batch, the tensorrt output is all [0,0,0,0,.....]

Tried dynamic batch, failed. Both onnx model and tensorRT engine can not output the expected output. Maybe this issue has some relationship with the implementation of Swin Transformer. Not sure.

**Comment by wangjingg** [comment]
When I cancel the softmax, in pytorch->onnx(dynamic batch),  It's work, you can try it,

**Comment by maggiez0138** [comment]
> When I cancel the softmax, in pytorch->onnx(dynamic batch), It's work, you can try it,

Thanks for your info, will do the attempt.

**Comment by fatemebafghi** [comment]
Hello there. I have the exact same problem with dynamic inputs, but I could not find a way to cancell softmax during converting pytorch model to onnx. Can you help me through this?

**Comment by maggiez0138** [comment]
> Hello there. I have the exact same problem with dynamic inputs, but I could not find a way to cancell softmax during converting pytorch model to onnx. Can you help me through this?

Updated the repo. Now dynamic input is supported.
